src/tdlib/chat.rs

```diff
@@ -481,6 +481,10 @@ impl Chat {
         self.type_().user() == Some(&self.session().me())
     }
 
+    pub(crate) fn is_replies_chat(&self) -> bool {
+        self.id() == 1271266957 || self.id() == 708513
```

Comment:
Does it have two ids?

Reply:
Because the second one is the id of the bot in the test backend.
This is also the same way Telegram Desktop implements it: https://github.com/telegramdesktop/tdesktop/blob/41d9a9fcbd0c809c60ddbd9350791b1436aff7d9/Telegram/SourceFiles/data/data_peer.cpp#L891

Reply:
@alissonlauffer Could one of the ids collide with a regular chat? I would suggest to use this
```rust
pub(crate) fn is_replies_chat(&self) -> bool {
        (self.session_().client_().database_info().0.use_test_dc && self.id() == 708513)
            || self.id() == 1271266957
}
```

Reply:
Well, actually, Marco suggested to handle `UpdateOption` instead (which includes a `replies_bot_chat_id` option), but I'm not sure how to correctly add support to tdlib options, since keys are just strings, and values can be of any type. Tdlib options are a nice thing to have in the session object, but it could be implemented in another PR, which could be merged before this one.

Reply:
You mean basically?
![grafik](https://github.com/paper-plane-developers/paper-plane/assets/3630213/800f36b7-956d-4012-bb08-69b69fcc6437)

Reply:
Yes, but we would need too much "boilerplate code" to do that, in my opinion.
But we can add tdlib options on demand also, there's no need to add all of them right away.

Reply:
> we can add tdlib options on demand also.

What do you mean?

Reply:
I mean, right now we would only require `replies_bot_chat_id`, but in the future, if we need more options, we could add one by one. But yeah, this could become an excuse for not using them.

Reply:
Yes, at Some point we should start using them. :laughing: 
I think using this option is the correct future-proof way, as we never know whether telegram might suddenly change such a magic id.